hr_employee_type_private_wizard/models/res_partner.py

```diff
@@ -0,0 +1,63 @@
+# © 2019 Numigi (tm) and all its contributors (https://bit.ly/numigiens)
+# License LGPL-3.0 or later (http://www.gnu.org/licenses/lgpl).
+
+from odoo import fields, models, _
+from odoo.exceptions import AccessError
+from odoo.osv.expression import AND, OR
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Unused AND imported from odoo.osv.expression](https://app.codacy.com/manual/numigi/odoo-hr-addons/pullRequest?prid=5358386)